pbspro.spec

```diff
@@ -466,6 +485,10 @@ fi
 %endif
 %exclude %{pbs_prefix}/unsupported/*.pyc
 %exclude %{pbs_prefix}/unsupported/*.pyo
+%exclude %{pbs_prefix}/lib/*.a
+%exclude %{pbs_prefix}/include/*
+%exclude %{pbs_prefix}/lib/*.so*
+%exclude %{pbs_prefix}/lib/pkgconfig/pbs.pc
```

Comment:
I think you only want to exclude the header files and archive libraries. The shared libraries and pbs.pc should not be excluded. Question: why is it not pbspro.pc?